tests/bluetooth/df/connectionless_cte_chains/src/common.c

```diff
@@ -440,6 +467,12 @@ void common_validate_chain_with_cte(struct ll_adv_set *adv, uint8_t cte_count,
 		if (idx < ad_data_pdu_count) {
 			ext_hdr_flags |= ULL_ADV_PDU_HDR_FIELD_AD_DATA;
 		}
+
+		if (IS_ENABLED(CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT) &&
+		    ext_hdr_flags != ULL_ADV_PDU_HDR_FIELD_NONE) {
+			ext_hdr_flags |= ULL_ADV_PDU_HDR_FIELD_ADI;
+		}
```

Comment:
In the three previous instances of `IS_ENABLED(CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT)` we don't check for `ext_hdr_flags != ULL_ADV_PDU_HDR_FIELD_NONE)`, but we do here. How come?

Reply:
@Thalley Fixed, ADI field shall be checked if `CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT=y` where superior PDU has ADI present. It is incorrect to have `ext_hdr_flags != ULL_ADV_PDU_HDR_FIELD_NONE` check.